.github/workflows/publish-site.yaml

```diff
@@ -19,38 +19,43 @@ jobs:
       pull-requests: 'write'
 
     steps:
-    - uses: actions/checkout@v4
-
-    - uses: actions/setup-node@v4
-      with:
-        cache: 'npm'
-        node-version: 20
-
-    - name: Setup Hugo
-      uses: peaceiris/actions-hugo@v3
-      with:
+      - uses: actions/checkout@v4
+        name: Checkout
+
+      - uses: actions/setup-node@v4
+        name: Setup Node.js
+        with:
+          cache: 'npm'
+          node-version: 20
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v3
+        with:
           hugo-version: 'latest'
 
-    - run: npm install
+      - run: npm install
+        name: Install Dependencies
 
-    - run: npm run build
+      - run: npm run build
+        name: Build Site
 
-    - name: Generate Schema
-      uses: actions/setup-python@v5
-      with:
-       python-version: '3.11'
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
 
-    - run: |
-        python3 -m pip install json-schema-for-humans
-        generate-schema-doc --config-file generate-schema.yaml static/schemas/component-descriptor-v2 schema-v2.html
-        generate-schema-doc --config-file generate-schema.yaml static/schemas/component-descriptor-v3alpha1 schema-v3alpha1.html
+      - run: |
+          python3 -m pip install json-schema-for-humans
+          generate-schema-doc --config-file generate-schema.yaml static/schemas/component-descriptor-v2 schema-v2.html
+          generate-schema-doc --config-file generate-schema.yaml static/schemas/component-descriptor-v3alpha1 schema-v3alpha1.html
+        name: Generate Schema
```

Comment:
is this line intended?

Reply:
in the diff overview the indent looks wrong and the name attribute not correctly colored (looks a bit like seen as part of the run command), but in the standard editor it was fine. I placed the name attribute at the top and now it also looks of in the diff overview.